src/Mapping/ClassMetadataFactory.php

```diff
@@ -710,6 +701,18 @@ private function inheritIdGeneratorMapping(ClassMetadata $class, ClassMetadata $
     protected function wakeupReflection(ClassMetadataInterface $class, ReflectionService $reflService): void
     {
         $class->wakeupReflection($reflService);
+
+        if (PHP_VERSION_ID < 80400) {
+            return;
+        }
+
+        foreach ($class->propertyAccessors as $propertyAccessor) {
```

Comment:
`ClassMetadataFactory:: validateRuntimeMetadata` is called to early, as it does not have access to any accessors yet. However this is in persistence and cannot be changed easily.